motoko/phone-book/src/declarations/index.js

```diff
@@ -0,0 +1,32 @@
+import { Actor, HttpAgent } from "@dfinity/agent";
```

Comment:
Why is this file and all other files in declarations checked in? Aren't they generated by the build process?